strategy/thompson_sampling_strategy.py

```diff
@@ -0,0 +1,152 @@
+"""
+
+ThompsonSampling Strategy
+
+"""
+
+import numpy as np
+import scipy.stats
+from copy import deepcopy
+from strategy.strategy import Strategy
+
+
+class ThompsonSampling(Strategy):
+    """ Thompson Sampling Strategy
+
+    Args:
+        bandit (Bandit): bandit
+        **kwargs : prior parameters
+
+    Attributes:
+        num_arms (int)
+        posterior_params (list of list): posterior parameters
+        estimated_arm_means (ndarray): posterior predictive mean
```

Comment:
I think they are the same in this case, since the posterior mean of the mean is also the posterior predictive mean. And we want to maximize the posterior predictive mean. 
The variance/standard deviation is another story. The posterior variance of the mean is smaller than the posterior predictive variance